### PR TITLE
Document admin aspect of installing the Media Viewer app

### PR DESCRIPTION
**Pull request by settermjd**
Relates to #1465, specifically https://github.com/owncloud/docs/issues/1465#issuecomment-522988656.

### CODE REVIEW

File: modules/admin_manual/pages/installation/apps/mediaviewer/index.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The {media-viewer-app-url}[Media Viewer App] is a simple viewer for pictures and videos integrated into the files app.
+
+== Requirements
+
+* {nodejs-url}[Node.js]
+* {npm-url}[NPM]
+
+== Installing
+
+Before installing the Media Viewer app, run `npm install` && `npm run build` to build the Media Viewer app's dependencies.
```
Comment (ho4ho) [inline modules/admin_manual/pages/installation/apps/mediaviewer/index.adoc]: Required by end-user? Not only possible to install from marketplace without node/npm?

Comment (settermjd) [reply modules/admin_manual/pages/installation/apps/mediaviewer/index.adoc]: This is the admin docs, so I don't see any issue with including it; especially as it's listed in [the repository's README](https://github.com/owncloud/files_mediaviewer).

**Comment by settermjd** [comment]
@pmaier1, can you review, please?

File: modules/admin_manual/pages/installation/apps/mediaviewer/index.adoc (hunk 1 above)
Comment (michaelstingl) [reply modules/admin_manual/pages/installation/apps/mediaviewer/index.adoc]: @settermjd no, please remove. Instructions in the README.md are for developers. Marketplace install or install with *.tar are the recommended ways.

Comment (ho4ho) [reply modules/admin_manual/pages/installation/apps/mediaviewer/index.adoc]: > This is the admin docs, so I don't see any issue with including it; especially as it's listed in the repository's README.

installing not same as building, README only mentions requirements for building for developers which is developer docs not admin docs.

Comment (settermjd) [reply modules/admin_manual/pages/installation/apps/mediaviewer/index.adoc]: Thank you for the feedback.

**Comment by settermjd** [comment]
@pmaier1, can you review, please?

**Comment by pmaier1** [comment]
As mentioned in the original ticket https://github.com/owncloud/docs/issues/1465
- Media Viewer supersedes the former Gallery and Video Player. Those at least need to be disabled for Media Viewer to work properly. It is recommended to remove them completely.
- Gallery and Video Player are not supported anymore and will not receive any further security or bug fixes. It's recommended to switch to the Media Player.
- Conversion of "old" album view public links https://github.com/owncloud/gallery#redirect-gallery-link-shares
- Link to Marketplace?

**Comment by settermjd** [comment]
> As mentioned in the original ticket #1465
> 
>     * Media Viewer supersedes the former Gallery and Video Player. Those at least need to be disabled for Media Viewer to work properly. It is recommended to remove them completely.
> 
>     * Gallery and Video Player are not supported anymore and will not receive any further security or bug fixes. It's recommended to switch to the Media Player.
> 
>     * Conversion of "old" album view public links https://github.com/owncloud/gallery#redirect-gallery-link-shares
> 
>     * Link to Marketplace?

These points have been addressed in the 6221a98  & 670c06b.

**Comment by pmaier1** [comment]
> These points have been addressed in the 6221a98 & 670c06b.

Do I just not see it or did you intentionally neglect the marketplace link?

**Comment by settermjd** [comment]
@pmaier1, it's right at the start of https://github.com/owncloud/docs/pull/1700/files#diff-304221cc77f03247de6989398085a251R7.

**Comment by pmaier1** [comment]
> @pmaier1, it's right at the start of https://github.com/owncloud/docs/pull/1700/files#diff-304221cc77f03247de6989398085a251R7.

`{media-viewer-app-url}` points to `https://github.com/owncloud/files_mediaviewer`

**Comment by settermjd** [comment]
> > @pmaier1, it's right at the start of https://github.com/owncloud/docs/pull/1700/files#diff-304221cc77f03247de6989398085a251R7.
> 
> `{media-viewer-app-url}` points to `https://github.com/owncloud/files_mediaviewer`

Sorry, my mistake. I'll get that fixed.

**Comment by settermjd** [comment]
> > @pmaier1, it's right at the start of https://github.com/owncloud/docs/pull/1700/files#diff-304221cc77f03247de6989398085a251R7.
> 
> `{media-viewer-app-url}` points to `https://github.com/owncloud/files_mediaviewer`

@pmaier1, the link's been changed in 7d32411e4d40f02a7ba08d01e530da42c4860526.

**Pull request merged**